packages/rspeedy/core/src/config/output/index.ts

```diff
@@ -335,4 +335,14 @@ export interface Output {
    * The {@link SourceMap} configures whether and how to generate source-map for outputs.
    */
   sourceMap?: boolean | SourceMap | undefined
+
+  /**
+   * The `debugIds` option controls whether to add debug IDs to the output files.
+   *
+   * @remarks
+   * When enabled, Rspeedy will add debug IDs to the output files to help with debugging.
+   * This is useful when you need to track down issues in production builds.
+   * See the {@link https://github.com/tc39/ecma426/blob/main/proposals/debug-id.md | TC39 sourcemap debug ID proposal} for more details.
+   */
+  debugIds?: boolean | undefined
```

Comment:
We already have the `output.sourceMap.js` option, I think it would be better to directly use `-debugids` here just like Rspack/Webpack does.

```js
export default {
  output: {
    sourceMap: { js: 'source-map-debugids' },
  },
}
```

> Note that we would like to keep the configuration as same as Rsbuild. So I'm not sure if adding `output.debugIds` is necessary.